components/shared_code/src/shared_data_model/parameters.py

```diff
@@ -177,6 +177,17 @@ class TargetBranchesToInclude(MultipleChoiceWithAdditionParameter):
     metrics: list[str] = ["merge_requests"]
 
 
+class BranchMergeStatus(MultipleChoiceParameter):
+    """Branch merge status."""
+
+    name: str = "Branch merge status"
+    short_name: str = "merge status"
+    help: str = "Limit which merge states to count."
+    placeholder: str = "all merge states"
+    metrics: list[str] = ["inactive_branches"]
+    values: list[str] = ["merged", "unmerged"]
```

Comment:
Should something like "merge with fast forward" or "merge with rebase" fit into this category? I guess that type of information can only be inferred from the pull/merge request and not the branch itself :(

Reply:
I guess so but yes, that would be more work to figure out and outside of the scope of this PR.